monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.sla

```diff
@@ -21964,13 +21964,13 @@
             <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="Kovri:"/>
             <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH="  Esta es una gran actualización de privacidad, agregando una cuarta capa de privacidad a las transacciones de Monero. Kovri utiliza "/>
             <ITEXT FONT="Adobe Caslon Pro Semibold" FONTSIZE="12" FEATURES="inherit underline" FCOLOR="quickfacts" CH="encriptación-garlic"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" (piensa en Tor con esteroides) para enmascar las direcciones IP de los remitentes y receptores utilizando la red de Monero. Cada capa de la transacción de Monero, desde el remitente, receptor, la cantidad tranzada y (con Kovri) el pilar de la transacción, serán privadas y seguras con el estado del arte de la encriptación. "/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" (piensa en Tor con esteroides) para enmascar las direcciones IP de los emisores y receptores utilizando la red de Monero. Cada capa de la transacción de Monero, desde el emisor, receptor, la cantidad de transacciones y (con Kovri) el pilar de la transacción, serán privadas y seguras con el estado del arte de la encriptación. "/>
             <para LINESP="13.5"/>
             <para LINESP="13.5"/>
-            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="Monederos físicos(hardware):"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" Los reconocidos monederos de Ledger y Trezor, ambos, están trabajando para agregar a Monero (Ledger tiene soporte desde el monedero CLI v0.12.2 and la GUI v0.12.3). La comunidad de Monero esta financiando a un equipo de trabajo que están "/>
-            <ITEXT FONT="Adobe Caslon Pro Semibold" FONTSIZE="12" FEATURES="inherit underline" FCOLOR="quickfacts" CH="construyendo un monedero físico"/>
-            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" construido sobre las características distintivas de Monero. Ambos monederos físicos se anticipa que estarán operativos para el final del 2018. "/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="Monederos físicos (hardware):"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" Los reconocidos monederos de Ledger y Trezor, ambos, están trabajando para agregar a Monero (Ledger tiene soporte desde la versión v0.12.2 para monero-wallet-cli, y v0.12.3 para monero-wallet-gui). La comunidad de Monero está financiando a un equipo de trabajo que está "/>
```

Comment:
"La comunidad de Monero está financiando a un equipo de trabajo que **esta**" without tilde.

[esta,ésta o está](https://www.diccionariodedudas.com/esta-esta-o-esta/) _**Esta**_ es un demostrativo femenino que puede referirse a lo que está cerca de la persona que habla, a algo que ha sido mencionado o será mencionado a continuación, o a un periodo de tiempo, un momento o un lugar que es próximo. No debe acentuarse en ningún caso.

Reply:
I will fix the original sentence I suggested, noticed that there are some ambiguitys when reading the text. I'm giving options with arguments. I suggest to do some research and share it inside the comments so we can continue with the workflow. @BlackLotus64 it seems you are straightforward in what you are suggesting on changing. 

"La comunidad de Monero **esta** financiando a un equipo de trabajo que **está**"

- Los determinantes demostrativos nunca han sido tildados: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa. Demonstrative determinants have never been labeled: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
- **Está** es la forma verbal en tercera persona del presente del modo indicativo del verbo **estar**. **Está** is the verbal form in the third person of the present in the indicative way of the verb **estar**.

This is an answer given inside a Peruvian university forum from someone asking for help in the issue for the word "esta, ésta, or está".

[Esta, está y ésta](http://udep.edu.pe/castellanoactual/duda-resuelta-esta-esta-y-esta/)

Answer:
>Estimada Rocío:
Una de las novedades de la Ortografía de la lengua española (2010) fue justamente la eliminación de la tilde diacrítica en los demostrativos cuando funcionan como pronombres: Estas se guardan en el cajón de arriba; *Éstas se guardan en el cajón de arriba.
Los determinantes demostrativos nunca han sido tildados: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
El demostrativo esta se usa para identificar a algo o a alguien que está cerca del hablante, pero lejos del oyente: Alumnos, observen con atención estas imágenes. Puede encontrar más información sobre este asunto en http://udep.edu.pe/castellanoactual/duda-resuelta-demostrativos/
**Está** es la forma verbal en tercera persona del presente del modo indicativo del verbo **estar**. Y se tilda porque es una palabra aguda terminada en vocal: El café está demasiado caliente.
Saludos cordiales,
Castellano Actual

Translation:
>One of the novelties of the Spelling of the Spanish language (2010) was precisely the elimination of the diacritic tilde in the demonstratives when they function as pronouns: Estas se guardan en el cajón de arriba; *Éstas se guardan en el cajón de arriba.
Demonstrative determinants have never been labeled: **Esta** cartera es preciosa; Mi mamá quiere vender **esta** casa.
The demonstrative is used to identify something or someone who is close to the speaker, but away from the listener: Alumnos, observen con atención estas imágenes. You can find more information about this matter at http://udep.edu.pe/castellanoactual/duda-resuelta-demostrativos/
**Está** is the verbal form in the third person of the present in the indicative way of the verb **estar**. And it is marked because it is a sharp word ending in a vowel: Coffee is too hot.
Best regards,
Current Spanish

Reply:
@BlackLotus64 ping

Reply:
Yes. As you stated, the word _está_ with a tilde is correct because in the next text like it talks about the Monero team developing a new wallet, so it is an action or, an indicative way.

Reply:
I suggest this:
"La comunidad de Monero **esta** financiando a un equipo de trabajo que está"

Removing the first "está" with tilde and leave the second one as you suggest.

@BlackLotus64

Reply:
Both should have the tilde since they are indicating actions and not a subject.

Reply:
ok